pages/es6_example/page.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Created by titusfortner on 11/23/16.
+ */
+class Page {
+
+    constructor(){ };
```

Comment:
Sorry, I'm a Javascript idiot, but does every class need to have an empty constructor explicitly defined?

Reply:
good catch -- see other comment about constructor passing browser